scripts/superdesk-search/react.js

```diff
@@ -390,6 +416,9 @@
                             return function(event) {
                                 event.stopPropagation();
                                 highlightsService.markItem(highlight._id, this.props.item);
+                                if (getViewType() === 'highlights') {
```

Comment:
In highlights only list view, removing the highlight of an item(single highlighted) removes that item from the list (unlike monitoring view item where just star removes). so only for viewType highlights we need to update the count if it's might be in multi-selection. however this should happen only if item is single highlighted. this check is also included in updated PR.